gpflow/likelihoods/multilatent.py

```diff
@@ -107,11 +107,14 @@ def __init__(
             function modelling the scale to ensure its positivity.
             Typically, `tf.exp` or `tf.softplus`, but can be any function f: R -> R^+.
```

Comment:
Could you add a line to the docstring explaining that the default is Exp if not explicitly specified?

Reply:
Done! Is it correct to say it defaults to `tf.exp` if not specified? The code says  `positive(base="exp")`, is that the same?

Reply:
It's de-facto the same; `positive()` constructs a tfp.bijectors.Bijector instance instead of returning a function, but both exponentiate the argument. Could just call it "exp" instead of referring to `tf.`...

Reply:
Ok, thanks for clarifying, did that.